app/models/player.rb

```diff
@@ -5,4 +5,46 @@ class Player < User
   has_many :coaches, through: :team_coaches
   has_one :profile, class_name: :PlayerProfile
   has_many :player_stats, through: :profile
+
+  def name
+    self.first_name + " " + self.last_name
+  end
+
+  def school
+    self.profile.school
+  end
+
+  def height
+    self.profile.height
+  end
+
+  def weight
+    self.profile.weight
+  end
+
+  def grade
+    self.profile.grade_level
+  end
+
+  def gpa
+    self.profile.gpa
+  end
+
+  def stats
+    self.player_stats
+  end
+
+  def points(x = 10)
+    Hash[self.stats.map{|stat| stat.stat_dates + "(game:#{stat.id})"}.zip(self.stats.pluck(:points))].to_a.reverse[0..x-1]
```

Comment:
@thedanielvogelsang 
Hey Dan,

Can you give us a quick summary of what this method (and by proxy, the fouls method) is doing?

Thanks!

Reply:
So this was just one hypothetical way to render season points-over-a-series of time (in this case the argument, x, is referencing how many 'games' (working backward from most recent) you'd like to return). 

I wrote this method in part in response to your total-points system that you had started on the teams page (though thereafter without much reference, so feel free to either tweak my method, or write your own). But I ultimately will be using the date and #points (or #fouls, as it were) as x,y coordinates on a graph on the player profile page.

To answer your question directly, this method takes an argument and pulls the last 'x' amount of dates and zips that together in a hash with those dates' correlating players' scored points (or fouls).